docs/how-to-add-new-function.md

```diff
@@ -85,8 +85,10 @@ public class SampleAlgo implements MLAlgo {
 }
 ```
 
+### (Optional)Step 5: register function object
+You can register instance of thread-safe class in ML plugin. Refer to Example2#Step5.
```

Comment:
is the Function annotation at step 4 still required?

Reply:
Good catch. If register explicitly, don't need to add function annotation at step4. Will add this